packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -305,6 +312,18 @@ export default class ConfigParser {
             throw new Error('The --multi-run flag requires that either the --spec or --suite flag is also set')
         }
 
+        // remove glob-pattern from CLI spec arg before filtering
+        if (this._config.spec !== undefined && this._config.spec.length > 0 && this.isGlobPattern(this._config.spec[0])){
+            this._config.spec[0] = this.removeGlobPatternFromSpecCliArg(this._config.spec[0])
+        }
```

Comment:
Why do we need to remove the glob pattern? Can we support it instead, e.g. allow `foo*` to match files like `foobar` or `foobarloo`?